metrics/errors.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  name: 'enables JavaScript Sentry error metrics',
+  startDate: '2015-01-01',
+  subjectAttributes: ['uuid', 'sampleRate'],
+  independentVariables: ['sentryEnabled'],
+  eligibilityFunction: function (subject) {
+    return !! subject.sampleRate && subject.uuid && this.bernoulliTrial(subject.sampleRate, subject.uuid)
```

Comment:
> Sample rate depends on the environment

Defining the `sampleRate` outside of the experiment violates the "spirit" of the experimenter being in control. There's no way for a person reading only this code to have any idea of how many subjects will be eligible.

I think a better way to handle this is exactly in @vladikoff's explanation "Sample rate depends on the environment". Let *that* be the input.

```js
var sampleRate = subject.env === 'dev' ? 1.0 : 0.3
```

Its a subtle difference, but now the experiment has full control again. The subject shouldn't be making decisions about the experiment, only providing enough data for the experiment to make a decision. Make sense?